data/data/bootstrap/files/usr/local/bin/bootkube.sh.template

```diff
@@ -64,7 +64,8 @@ then
 		"${RELEASE_IMAGE_DIGEST}" \
 		render \
 			--output-dir=/assets/cvo-bootstrap \
-			--release-image="${RELEASE_IMAGE_DIGEST}"
+			--release-image="${RELEASE_IMAGE_DIGEST}" \
+			--cluster-profile={{.ClusterProfile}}
```

Comment:
The CVO render gets cluster-profile parameter?
I thought it get it as env - https://github.com/openshift/cluster-version-operator/pull/483#issue-528097102